rpmvenv/extensions/core.py

```diff
@@ -59,6 +60,16 @@
             description='The build architecture to use.',
             required=False
         ),
+        requires=ListOption(
+            option=StringOption,
+            description='Dependencies',
+            required=False
+        ),
+        provides=ListOption(
+            option=StringOption,
```

Comment:
I think this is something I could make much clearer in the confpy documentation:

The `option` parameter of `ListOption` actually needs to be a configured instance of an option rather than the type.

```python
ListOption(
    option=StringOption(),
    ...
)
```